src/test/java/teammates/test/cases/browsertests/FeedbackRankQuestionUiTest.java

```diff
@@ -122,6 +122,47 @@ public void testStudentSubmitAndResultsPages() throws Exception {
         // to verify that the question submit form with existing response works correctly
         submitPage.verifyHtmlMainContent("/studentFeedbackSubmitPageSuccessRank.html");
 
+        ______TS("Rank : min/max options to be ranked test");
+
+        // Testing question with only min options to be ranked restriction
```

Comment:
I don't think we start other similar comments with `Testing`. 
`// Question with only min options to be ranked restriction` ?